sns_ik_lib/src/sns_position_ik.cpp

```diff
@@ -53,8 +53,8 @@ int SNSPositionIK::CartToJnt(const KDL::JntArray& joint_seed,
   // TODO: use tolerance twist
   double linearTolerance = 1e-5;
   double angularTolerance = 1e-5;
-  double linearMaxStepSize = 0.05;
-  double angularMaxStepSize = 0.05;
+  double linearMaxStepSize = 0.2;
+  double angularMaxStepSize = 0.2;
```

Comment:
I actually had worse results on my workstation for `angular` step size = `0.2`. Shall we leave it the same?